googletest/src/internal/test_outcome.rs

```diff
@@ -246,3 +260,34 @@ impl From<TestAssertionFailure> for proptest::test_runner::TestCaseError {
         proptest::test_runner::TestCaseError::Fail(format!("{value}").into())
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Could you do a bit more testing:

- Check whether we can / should update the error message verification in `test_can_return_anyhow_generated_error()` in integration_tests/src/integration_tests.rs

- Add a new integration test that returns a Result with a custom error type like integration_tests/src/test_returning_anyhow_error.rs